src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -60,6 +63,11 @@ private[query] class EventsByTagPublisher(
     case None       => Long.MaxValue
   }
 
+  Try { // Ignore pubsub when clustering unavailable
+    DistributedPubSub(context.system).mediator !
+      DistributedPubSubMediator.Subscribe("akka.persistence.cassandra.journal.tag", self)
+  }.filter(_ => eventualConsistencyDelayMillis == 0)
```

Comment:
It was supposed to only register when eventualConsistencyDelayMillis is not 0. It doesn't :-)

I'll rewrite it as a simple `if`.